### ISSUE TITLE
How use Mediapipe to extract position of landmarks and bounding boxe

### ISSUE THREAD

**Issue by dgrnd4** [madelinegannon/example-mediapipe-udp#9]
Could you please tell me how to extract those position, using mediapipe?
Many thanks

**Comment by madelinegannon** [comment]
Hi @dgrnd4 ... The [my_pass_through_calculator.cc](https://github.com/madelinegannon/example-mediapipe-udp/blob/master/mediapipe/my_pass_through_calculator.cc#L158) file shows how you access different features by tags (like `"hand_landmarks"`, `"palm_dectections"`, `"hand_rect"`) with mediapipe.

**Comment by dgrnd4** [comment]
i have the default code of mediapipe and i want to get this information but honestly i dont understand those things:
1) must i change all the codes as yours?
if not the 1 -> 2) which file should i modify? The demo_run_graph_main.cc or something else?

i just want to print in a .txt file something like this:
landmarks: positions , bounding boxe: positions

**Comment by madelinegannon** [comment]
Yeah, I feel you ... MediaPipe's modular graph-based architecture is not particularly intuitive. I'm still discovering new things about it, too.

In this repo/tutorial, I created a new node (`my_pass_through_calculator.cc`) where I formatted the hand data however I wanted, and then I integrated it into the compute graph for one of their example projects (in my case, this was their `hand_tracking` demo). You could do something similar, but just print out the data instead of sending out over UDP.

Have you worked your way through MediaPipe's [Getting Started](https://google.github.io/mediapipe/getting_started/getting_started.html) documentation? There's a [Hello World!](https://google.github.io/mediapipe/getting_started/hello_world_desktop.html) example that is helpful. Also there's more information on [how to get help](https://google.github.io/mediapipe/getting_started/help.html) from the MediaPipe team.

**Comment by dgrnd4** [comment]
Many thanks, i get it using code picked up from the mediapipe original directory to extract the position of the landmarks.

Now i need the position of the boxe so i think that i'll use you code of the my_pass_through_calculator. Many thanks

**Comment by dgrnd4** [comment]
i'm reading now that you are using hand_tracking while i need multi_hand_tracking.
Do you have any solution for this?

**Comment by dgrnd4** [comment]
i'm reading now that you are using hand_tracking while i need multi_hand_tracking.
Do you have any solution for this?
